docs/developer-documentation.md

```diff
@@ -85,6 +85,24 @@ Check the [UI Patterns documentation][2] for more information about how to use p
 
 Below a list of available value object along with its factory methods:
 
+#### `DateValueObject`
+
+Used in the following patterns:
+
+- [`date_block`](../templates/patterns/date_block/date_block.ui_patterns.yml)
+
+Provides the following factory methods:
+
+- `DateValueObject::fromArray(array $values = [])`: accepts an array with the following properties:
```

Comment:
The PR is fine, but do we really need this static method? Its essentially the same as the fromTimestamp method but a bit worse since fromTimestamp tells you what parameters to pass.

Reply:
We need this, yes. It's defined in the the interface... and we cannot go over it.